### DISCUSSION TITLE
Can not copy or move files between external drives NC AIO

### DISCUSSION THREAD

**Discussion by richardcrbb** [nextcloud/all-in-one#4371] [Questions] [answered]
Hi all, I’ve got a problem with my external drives on my NC AIO v7.13.1, I am trying to move files from one drive to the other.

Hardward RPI5
HDD 2.5 toshiba 1tb dtb410 with two partitions nube1 and nube2
SSD m.2 Kingston NV2 with partition nube

Software
OS Ubuntu Server 23.10
NC AIO v7.13.1

When I try to copy or move files from my toshiba hhd partition nube1 to my kingston ssd partition nube I get the following error

[PHP] Error: rename(//mnt/nube1/My Photos History/paseo san andres 2024/GH015294.MP4,//mnt/nube/My Photos History/paseo san andres 2024/GH015294.MP4): Bad address at /var/www/html/lib/private/Files/Storage/Local.php#393

MOVE /remote.php/dav/files/admin/nube1/My%20Photos%20History/paseo%20san%20andres%202024/GH015294.MP4
from 201.244.140.49 by admin at 2024-03-12T13:11:41+00:00

I have my permission like this:

drwxr-x— 6 www-data root 4096 Mar 12 12:04 ncdata
drwxr-x— 5 www-data root 4096 Mar 11 21:04 nube
drwxr-x— 3 www-data root 4096 Feb 29 20:56 nube1
drwxr-x— 9 www-data root 4096 Feb 29 10:28 nube2

What am I doing wrong?

PLEASE I NEED SOME HELP!

**Comment by szaimen** [comment, marked as answer]
This is getting handled in https://help.nextcloud.com/t/can-not-copy-or-move-files-between-external-drives-nc-aio/185180
